sdk/scheduler/src/test/resources/invalid-network.yml

```diff
@@ -0,0 +1,13 @@
+name: "docker-test"
+pods:
+  server:
+    count: 1
+    networks:
```

Comment:
YAML allows comments which is nice.  It's not immediately obvious to me why this is invalid, and I work with this stuff a lot.  Maybe a comment wouldn't go astray.